charts/icm-as/values.yaml

```diff
@@ -154,16 +154,18 @@ operationalContext:
   # name of the environment (environment-type + some sort of id, e.g. int-01, default=<environmentType>)
   environmentName: prd
 
-# provide custom cluster config from "icm-as.fullname"-system-conf-cluster configmap
-provideCustomConfig:
```

Comment:
May deprecate this configuration instead, however removing it is an breaking change, requiring a major release.

Reply:
Nobody is using this.